README.md

```diff
@@ -41,6 +41,99 @@ root.mainloop()
 ```
 
 
+## Tips and tricks
+Sun Valley is meant to be a simple ttk theme wrapped in a PyPI package for convenience. I don't want to add other dependencies, nor features that would be trivial to implement if the user wants them, using 3rd party modules. But this theme can be easily integrated with them.
+
+<details>
+  <summary><b>Set the theme to the system theme</b></summary>
+  
+  You can use [darkdetect](https://github.com/albertosottile/darkdetect) for that. Here's an example:
+  
+  ```python
+  import tkinter, sv_ttk, darkdetect
+  from tkinter import ttk
+
+  root = tkinter.Tk()
+
+  button = ttk.Button(root, text="Click me!", style="Accent.TButton")
+  button.pack()
+
+  sv_ttk.set_theme(darkdetect.theme())
+  root.mainloop()
```

Comment:
```suggestion
  import darkdetect

  sv_ttk.set_theme(darkdetect.theme())
```

I'd like to emphasize the simplicity of adding this functionality, and not repeat the example that is already presented above.